src/QueryBuilder/QueryBuilder/helpers/selectOptions.test.js

```diff
@@ -475,3 +475,19 @@ describe('getFilteredOptions', () => {
     expect(res).toEqual([{ label: '項目 - 持股 - 報表' }]);
   });
 });
+
+describe('getOperatorType', () => {
+  test('should return the correct operator group for an operator', () => {
+    expect(getOperatorType(OPERATORS.EQUAL)).toBe('comparison');
+    expect(getOperatorType(OPERATORS.NOT_EQUAL)).toBe('comparison');
+    expect(getOperatorType(OPERATORS.CONTAINS)).toBe('like');
+    expect(getOperatorType(OPERATORS.STARTS_WITH)).toBe('like');
+    expect(getOperatorType(OPERATORS.EMPTY)).toBe('null');
+    expect(getOperatorType(OPERATORS.IN)).toBe('arrayCompression');
+    expect(getOperatorType(OPERATORS.NOT_IN)).toBe('arrayCompression');
```

Comment:
I think you mean `arrayComparison`? In the end this value is just used in a string-comparison so it could just as well be a random string, but if you want a meaningful key-name, I think `arrayComparison` is a better option than `arrayCompression`.

Reply:
got it! Fixed typo